packages/contracts/contracts/plugin/PluginRepo.sol

```diff
@@ -150,11 +153,29 @@ contract PluginRepo is
     //         contentURI: cid12
     // //  },
 
+    /// @inheritdoc IPluginRepo
+    function updateReleaseMetadata(
+        uint8 _release,
+        bytes calldata _releaseMetadata
+    ) external auth(address(this), CREATE_VERSION_PERMISSION_ID) {
```

Comment:
Maybe we change the permission name in such logically, it relates to both version creationa nd metadata update.

Reply:
amended